Packs/CommonScripts/Scripts/script-ExampleJSScript.yml

```diff
@@ -65,7 +65,7 @@ script: >-
 
   // Util function http
 
-  var res = http('http://www.demisto.com');
+  var res = http('http://www.paloaltonetworks.com');
```

Comment:
```suggestion
  var res = http('https://www.paloaltonetworks.com');
```

Reply:
@Itay4 I rather keep it `http` in order to not depend on the site certificates